.gitignore

```diff
@@ -1,14 +1,2 @@
-build/
-node_modules/
-.DS_Store
```

Comment:
Non-blocking: Should we keep the `.DS_Store` so it is not accidentally merged when working in MacOS?

Reply:
Nice catch! Will ignore it. Been in Linux-land too long :stuck_out_tongue:

Reply:
Fixed in 445d20b

Reply:
An alternative POV shared with me when I contributed to the uniswap repo: `.DS_Store` is an OS-wide thing not unique to this project. Therefore, they require it to be a part of the global `.gitignore` file on their developer machines to not pollute the project's gitignore.